packages/schemas/features/blocks/bubbles/video/constants.ts

```diff
@@ -17,6 +17,8 @@ export const defaultVideoBubbleContent = {
   height: 400,
   aspectRatio: '16/9',
   maxWidth: '100%',
+  areControlsDisplayed: true,
+  isAutoplayEnabled: false,
```

Comment:
Should be `true` by default

Reply:
Updated


https://github.com/baptisteArno/typebot.io/assets/69730155/917386b8-66ae-45fc-b12d-06c4cfeb86ed